src/components/App.js

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
-import Display from './Display';
+import React, { useState } from 'react';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../logic/calculate';
+import Display from './Display';
+
+const App = () => {
+  const [total, updateTotal] = useState('0');
+  const [next, updateNext] = useState('0');
+  const [operation, updateOperation] = useState('');
+
+  const handleClick = (btnName) => {
+    const obj = { total, next, operation };
+    const calcObj = calculate(obj, btnName);
+    updateTotal(calcObj.total);
+    updateNext(calcObj.next);
+    updateOperation(calcObj.operation);
+  };
 
-export default function App() {
-  calculate();
   return (
     <>
-      <Display result="0" />
-      <ButtonPanel />
+      <div>
+        <Display result={next.toString()} />
```

Comment:
- [ ] [REQUIRED] it passes the `total` or `next` value from state to `Display` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passToatalToDisplay.png))